composer.json

```diff
@@ -22,11 +22,10 @@
     "require": {
         "php": ">=7.2",
         "ext-json": "*",
-        "automattic/jetpack-connection": "2.8.2",
-        "automattic/jetpack-config": "1.15.2",
-        "automattic/jetpack-autoloader": "2.11.18",
-        "automattic/jetpack-identity-crisis": "0.19.0",
-        "automattic/jetpack-sync": "2.16.3",
+        "automattic/jetpack-connection": "2.12.4",
+        "automattic/jetpack-config": "2.0.4",
+        "automattic/jetpack-autoloader": "3.0.10",
```

Comment:
These changes are related to major versions. But the breaking changes here are related to supporting PHP >= 7.0, which has been supported for a long time for WooPayments. 

- https://github.com/Automattic/jetpack-config/blob/trunk/CHANGELOG.md#200---2023-11-20
- https://github.com/Automattic/jetpack-autoloader/blob/trunk/CHANGELOG.md#300---2023-11-20